test/e2e/fixtures/preprod/get-protocol-parameters.e2e.ts

```diff
@@ -0,0 +1,70 @@
+import { expect } from 'vitest';
+
+export default [
+  {
+    testName: 'GET_PROTOCOL_PARAMETERS success - preprod',
+    result: {
+      data: {
+        epoch: expect.any(Number),
+        min_fee_a: 44,
+        min_fee_b: 155_381,
+        max_block_size: 90_112,
+        max_tx_size: 16_384,
+        max_block_header_size: 1100,
+        key_deposit: '2000000',
+        pool_deposit: '500000000',
+        e_max: 18,
+        n_opt: 500,
+        a0: 0.3,
+        rho: 0.003,
+        tau: 0.2,
+        decentralisation_param: 0,
+        extra_entropy: null,
+        protocol_major_ver: 9,
```

Comment:
```suggestion
        protocol_major_ver: 10,
```

looks like it's on 10 now